arkouda_proxy_server/src/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "src"
+version = "0.1.0"
```

Comment:
Again should this match the version in `arkouda_proxy_server/version.py`?

Reply:
@Ethan-DeBandi99 yup, fixed